sample/mpp-library/src/commonMain/kotlin/com/icerockdev/library/PermissionsHandler.kt

```diff
@@ -0,0 +1,45 @@
+package com.icerockdev.library
+
+import dev.icerock.moko.permissions.Permission
+import dev.icerock.moko.permissions.PermissionsController
+import dev.icerock.moko.permissions.DeniedException
+import dev.icerock.moko.permissions.DeniedAlwaysException
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
+import kotlin.coroutines.CoroutineContext
+
+expect val mainCoroutineContext: CoroutineContext
+
+class PermissionsHandler(
```

Comment:
replace it to some `TestViewModel` with moko-mvvm usage...to simplify sample

Reply:
Was replaced by ViewModel.